mlir/lib/Dialect/Polynomial/IR/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+set(LLVM_TARGET_DEFINITIONS PolynomialCanonicalization.td)
+mlir_tablegen(PolynomialCanonicalization.inc -gen-rewriters)
+add_public_tablegen_target(MLIRPolynomialCanonicalizationIncGen)
```

Comment:
Don't you need to depend on MLIRPolynomialCanonicalizationIncGen somewhere?

Reply:
+1. Bazel has such a dependency.

Reply:
In principle I agree (added in 0e25a328d9d05c080bef4e81df276b4f5a2b3a77).

In practice I am baffled by how CI and my local tests are passing without this dependency added. Excuse my build system naïveté, but maybe someone could help me understand why? Without that, I can't even be 100% sure I added the dependency in the right place.

Reply:
Most likely it would be concurrency and luck.